pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -364,7 +364,8 @@ func (configgen *ConfigGeneratorImpl) buildSidecarInboundListeners(
 			}
 
 			pluginParams := &plugin.InputParams{
-				ListenerProtocol:           plugin.ModelProtocolToListenerProtocol(node, endpoint.ServicePort.Protocol),
+				ListenerProtocol: plugin.ModelProtocolToListenerProtocol(node, endpoint.ServicePort.Protocol,
```

Comment:
I remember during our offline chat you mentioned the inbound filter chain match case 2, which is TLS + TCP, you changed from alpn=[istio] to alpn=[].
Is here the key to revert the change?

Reply:
This PR is for add flag for adding missing flag for inbound listener only.

Reply:
If the flag can fully turn off the inbound listener I am fine.